src/ActivityLogger.php

```diff
@@ -104,6 +104,18 @@ public function withProperty(string $key, $value)
         return $this;
     }
 
+    public function createdAt($dateTime)
+    {
+        $this->getActivity()->created_at = $dateTime;
+
+        return $this;
+    }
+
+    public function at($dateTime)
```

Comment:
I know we do this with all other methods but here I think that the conflict potential and misunderstanding is too big.